rebuildability_check.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+import argparse
+import logging
+import sys
+
+import osc.core
+import osc.conf
+
+import osclib.remote_project
+from osclib.remote_project import RemoteProject
+
+class RebuildabilityChecker(object):
+    def __init__(self, project_str):
+        self.logger = logging.getLogger('RebuildibilityChecker')
+        self.project = RemoteProject.find(project_str) # apiurl should be read from osc.conf.config['apiurl'], osclib config class looks like has different goal?
```

Comment:
that might be just me but could you limit your line length to a value that makes reading the comments possible without scrolling left and right on github page? That would be awesome

Reply:
goal of this comments is mainly discussion about code where we cooperate. It should not be there before we merge it.
But thanks for catch